src/main/resources/templates/db_scripts/populate-rules.sql

```diff
@@ -0,0 +1,97 @@
+      "name": "alphabetical_letters",
+      "type": "RegExp",
+      "validationType": "Strong",
+      "state": "Enabled",
+      "moduleName": "mod-password-validator",
+      "expression": "(?=.*[a-z])(?=.*[A-Z]).+",
+      "description": "The password must contain both upper and lower case letters",
+      "orderNo": 1,
+      "errMessageId": "password.alphabetical.invalid"
+    }'),
+('3e3c53ae-73c2-4eba-9f09-f2c9a892c7a2', '{
+      "ruleId": "3e3c53ae-73c2-4eba-9f09-f2c9a892c7a2",
+      "name": "numeric_symbol",
+      "type": "RegExp",
+      "validationType": "Strong",
+      "state": "Enabled",
+      "moduleName": "mod-password-validator",
+      "expression": "(?=.*\\d).+",
+      "description": "The password must contain at least one numeric character",
+      "orderNo": 2,
+      "errMessageId": "password.number.invalid"
+    }'),
+('2e82f890-49e8-46fc-923d-644f33dc5c3f', '{
+      "ruleId": "2e82f890-49e8-46fc-923d-644f33dc5c3f",
+      "name": "special_character",
+      "type": "RegExp",
+      "validationType": "Strong",
+      "state": "Enabled",
+      "moduleName": "mod-password-validator",
+      "expression": "(?=.*[!\"#$%&''()*+,-./:;<=>?@\\[\\]^_`{|}~]).+",
+      "description": "The password must contain at least one special character",
+      "orderNo": 3,
+      "errMessageId": "password.no-special-character"
+    }'),
+('2f390fa6-a2f8-4027-abaf-ee61952668bc', '{
+      "ruleId": "2f390fa6-a2f8-4027-abaf-ee61952668bc",
+      "name": "no_user_name",
+      "type": "RegExp",
+      "validationType": "Strong",
+      "state": "Enabled",
+      "moduleName": "mod-password-validator",
+      "expression": "^(?:(?!<USER_NAME>).)+$",
+      "description": "The password must not contain your username",
+      "orderNo": 4,
+      "errMessageId": "password.username.duplicate"
+    }'),
+('8d4a2124-8a54-4c49-84c8-36a8f7fc01a8', '{
+      "ruleId": "8d4a2124-8a54-4c49-84c8-36a8f7fc01a8",
+      "name": "sequence_characters",
+      "type": "RegExp",
+      "validationType": "Strong",
+      "state": "Enabled",
+      "moduleName": "mod-password-validator",
+      "expression": "^(?:(?!qwe)(?!asd)(?!zxc)(?!qaz)(?!zaq)(?!xsw)(?!wsx)(?!edc)(?!cde)(?!rfv)(?!vfr)(?!tgb)(?!bgt)(?!yhn)(?!nhy)(?!ujm)(?!mju)(?!ik,)(?!,ki)(?!ol.)(?!.lo)(?!p;/)(?!/;p)(?!123).)+$",
+      "description": "The password must contain at least one special character",
+      "orderNo": 5,
+      "errMessageId": "password.no-special-character"
+    }'),
+('98b961b4-16b8-4e62-a359-abf3805e16b0', '{
+      "ruleId": "98b961b4-16b8-4e62-a359-abf3805e16b0",
+      "name": "repeating_characters",
+      "type": "RegExp",
+      "validationType": "Strong",
+      "state": "Enabled",
+      "moduleName": "mod-password-validator",
+      "expression": "^(?:(.)(?!\\1))*$",
+      "description": "The password must not contain repeating symbols",
+      "orderNo": 6,
+      "errMessageId": "password.rule.repeating.symbols.invalid"
```

Comment:
Please change "password.rule.repeating.symbols.invalid" to the following format `password.<name>.invalid`. There is also a convention for translation keys on UI to use dots for nesting.

Reply:
Viktor, thank you for the notes. Will change it.